### PR TITLE
cmd: migrate 'surgery clear-page' command to cobra style command

### PR DESCRIPTION
**Pull request by ahrtr**
Linked to https://github.com/etcd-io/bbolt/issues/472

All surgery commands are migrated to cobra style commands!

### CODE REVIEW

**Comment by ahrtr** [comment]
cc @mitake @ptabor @serathius @spzala PTAL

After this PR gets merged, all surgery commands are migrated to cobra style commands!

**Comment by ahrtr** [comment]
ping @ptabor @serathius PTAL

File: cmd/bbolt/command_surgery.go [hunk 1]
```diff
@@ -159,6 +164,74 @@ func surgeryCopyPageFunc(srcDBPath string, cfg surgeryCopyPageOptions) error {
...
+	if o.pageId < 2 {
+		return fmt.Errorf("the pageId must be at least 2, but got %d", o.pageId)
+	}
+	return nil
+}
+
+func newSurgeryClearPageCommand() *cobra.Command {
+	var o surgeryClearPageOptions
+	clearPageCmd := &cobra.Command{
+		Use:   "clear-page <bbolt-file> [options]",
```
Comment (ptabor) [inline cmd/bbolt/command_surgery.go:190]: Shell we mention `--pageId` as mandatory option here ?

**Review by ptabor — APPROVED** [review]
Thank you !

File: cmd/bbolt/command_surgery.go (hunk 1 above)
Comment (ahrtr) [reply cmd/bbolt/command_surgery.go:190]: Yes, makes sense. We can use `cobra.MarkFlagRequired` to mark the field required. We should apply the same restriction for other surgery command as well. 

Let me do it as a followup. thx

**Pull request merged**